bindings/expose-friction-compensation.cpp

```diff
@@ -0,0 +1,39 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 2-Clause License
+//
+// Copyright (C) 2025, INRIA
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include <eigenpy/eigenpy.hpp>
+#include <pinocchio/multibody/fwd.hpp>
+
+#include "simple-mpc/friction-compensation.hpp"
+
+namespace simple_mpc
+{
+  namespace python
+  {
+    namespace bp = boost::python;
+
+    /* Eigen::VectorXd computeFrictionProxy(
+      FrictionCompensation & self,
+      Eigen::Ref<const VectorXd> velocity,
+      Eigen::Ref<VectorXd> torque) {
+        self.computeFriction(velocity, torque);
+
+        return torque;
+    } */
```

Comment:
Could you uncomment this proxy function.

I think passing a np array by reference in order to get the result of the computation is very confusing for python user, and will probably make the code harder to understand